test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -298,7 +298,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					dockerVersionCmd := fmt.Sprintf("\"docker version\"")
 					for _, n := range nodes {
 						if n.IsWindows() {
-							if eng.ExpandedDefinition.Properties.WindowsProfile != nil && !eng.ExpandedDefinition.Properties.WindowsProfile.SSHEnabled {
+							if eng.ExpandedDefinition.Properties.WindowsProfile != nil && !eng.ExpandedDefinition.Properties.WindowsProfile.GetSSHEnabled() {
```

Comment:
for example, the method invocation here will operate against a ContainerService object post-template generation (i.e., we can rely upon the actual data in the object and not have to worry about highjacking the underlying value w/ the default const value)

Reply:
What about the case where we are working with an template generated with an older version of aks-engine which may not have a value for sshEnabled stored? 
If we call GetSSHEnabled() we are guaranteed to never have a nil value.

Reply:
Good point. I think we have the runner call a func/method in its own area that wraps nil values underneath appropriate defaults (true/false)